assets/scripts/index.js

```diff
@@ -1,7 +1,14 @@
 'use strict';
 
-// user require with a reference to bundle the file and use it in this file
-// var example = require('./example');
+const setAPIOrigin = require('../../lib/set-api-origin');
+const config = require('./config');
+
+$(() => {
+  setAPIOrigin(location, config);
+});
+
+// use require with a reference to bundle the file and use it in this file
+// const example = require('./example');
 
 // use require without a reference to ensure a file is bundled
 require('./example');
```

Comment:
You can delete this line. I thought you did earlier when I was pairing with you?